src/frontends/tensorflow/src/op/sparse_reshape.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "helper_ops/sparse_fill_empty_rows.hpp"
+#include "helper_ops/sparse_segment_ops.hpp"
+#include "helper_ops/unique.hpp"
+#include "ngraph/validation_util.hpp"
+#include "op_table.hpp"
+#include "openvino/core/validation_util.hpp"
+#include "openvino/opsets/opset8.hpp"
+
+using namespace std;
+using namespace ov::opset8;
+using namespace ngraph;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_sparse_reshape_op(const NodeContext& node) {
+    // Currently, the translation for SparseReshape is possible only if new shape value is the same as the input shape
+    // value or it is different just by one dynamic dimension of the new shape that can be replace with the
+    // corresponding static dimension of the input shape.
+    default_op_checks(node, 3, {"SparseReshape"});
+    auto input_indices = node.get_input(0);
+    auto input_shape = node.get_input(1);
+    auto new_shape = node.get_input(2);
+
+    // TODO: need to provide the full constant folding for WD both new_shape inputs of SparseReshape
```

Comment:
can you remove commented code?

Reply:
No, that is a problem that I will solve in a different PR. It lacks evaluate method for some operation and for this reason the constant folding is not complete